modules/microshift-install-bootc-get-published-image.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-bootc-image.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-bootc-get-published-image_{context}"]
+= Getting the published bootc image for {microshift-short}
+
+Use the following procedure to find and use the {microshift-short} container images to install {op-system-base-full}.
+
+.Prerequisites
+
+* You have an x86_64 or AArch64 platform.
```

Comment:
You have access to `registry.redhat.io`?

Reply:
I was wondering why we didn't have prereqs here. Do we also need root access?

Reply:
No root access is required because we're only pulling images in this section.